src/OperatorMsg.chpl

```diff
@@ -148,6 +148,10 @@ module OperatorMsg
           when (DType.Bool, DType.Bool) {
             var l = toSymEntry(left,bool, nd);
             var r = toSymEntry(right,bool, nd);
+            if (op == "<<") || (op == ">>" )  {
+              var e = st.addEntry(rname, l.tupShape, uint);
+              return doBinOpvv(l, r, e, op, rname, pn, st);
```

Comment:
i don't think it matters since we're only shifting bools by bools, but I wanted to point out that we create `e` with `uint` type but we cast them to `int` in `doBinOp`

Reply:
Thanks for catching this.  I changed the type to int to better match numpy.  Numpy returns int(8), but when I tried to change to int(8) I received this error: 
RuntimeError: IOError Line 97 In MultiTypeSymbolTable.addEntry: The server was not configured to support pdarray's of type int(8). Please update the configuration and recompile.